api/src/main/java/org/pmiops/workbench/db/dao/UserServiceImpl.java

```diff
@@ -417,6 +418,41 @@ public DbUser createUser(
     return dbUser;
   }
 
+  /**
+   * Save updated dbUser object if requireInstitutionalVerification is enabled: Get the existing
+   * dbExistingVerifiedInstitutionalAffiliation and update it with Institution role and other text
+   *
+   * @param dbUser
+   * @param dbVerifiedAffiliation
+   * @return
+   */
+  @Override
+  public DbUser updateUser(
```

Comment:
Will change it to updateUserWithConflictHandling since i do want the name to specify that we are updating user and its related table (like verified institution in this case)